classes/04class/exercises/c04-iac01/frdvo/ec2.tf

```diff
@@ -0,0 +1,14 @@
+# Define a new resource of the EC2 instace type
+resource "aws_instance" "devops_academy_ec2" {
+  ami               = "ami-088ff0e3bde7b3fdf"
+  availability_zone = "ap-southeast-2a"
+  instance_type     = "t2.micro"
+  iam_instance_profile = "ec2-read-s3-role-tf-profile2"
```

Comment:
hum..this part will need changes.

the same way you leveraged internal references in terraform for `vpc_security_group_ids` just below, you need to apply here. 

so just point to the resource aws_iam_instance_profile you are creating in terraform.
https://github.com/devopsacademyau/academy/pull/1316/files#diff-5a4a5a16e11f5321bf4a6db967e19e4eR46-R50